### PR TITLE
Changed format of README and added screenshot

### PR DESCRIPTION
**Pull request by Sudoxo**
Changed format of README from .txt to .md, added headers, added screenshot. [GCI task]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
-Egg Chess v1.1 (010910) by Anders Lindstr�m
-============================================
+# Egg Chess v1.1 (010910) by Anders Lindstr�m
 
-1. Intro
+![EggChess screenshot](EggChess.png "EggChess")
+
+### 1. Intro
```
Comment (humdingerb) [inline README.md]: I'd remove that line...

File: README.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
 This is free software (GNU GPL), read COPYING for details.
```
Comment (humdingerb) [inline README.md]: ...and remove that one, too. There's no COPYING file currently, but we can add a LICENSE file. Anyway, no need to mention it in the readme.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
 This is a very simple game called Egg Chess (or in swedish: �ggaschack).
 
 
-1.1 Rules
+#### 1.1 Rules
```
Comment (humdingerb) [inline README.md]: Remove the "1.1" and level up the heading to `###`.

File: README.md [hunk 4]
```diff
@@ -18,7 +19,7 @@ The winner is the player who gets three whole, half eaten or empty eggs in
 a row.
 
 
-2. Installation
+### 2. Installation
```
Comment (humdingerb) [inline README.md]: Remove the "2."

File: README.md [hunk 5]
```diff
@@ -33,7 +34,8 @@ anywhere as long as the directory gfx/ is in the same place.
 Egg chess supports cross compiling from Linux to Win32.
```
Comment (humdingerb) [inline README.md]: Remove, doesn't apply to Haiku version.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Review by Sudoxo — COMMENTED** [review]
I made changes which you asked for

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks.

**Pull request merged**